doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -14086,6 +14107,375 @@ AirConditioner:VariableRefrigerantFlow,
 
 
 
+### AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl
+
+#### Field: Heat Pump Name 
+
+This alpha field defines a unique user-assigned name for an instance of a variable refrigerant flow heat pump. Any reference to this heat pump will use this name. Since this object is not listed in an AirloopHVAC object, the most likely use of this name would be for reporting purposes.
+
+#### Field: Availability Schedule Name
+
+This alpha field defines the name of the schedule (ref: Schedule) that denotes whether the heat pump operates during a given time period. A schedule value equal to 0 denotes that the heat pump must be off for that time period. A value other than 0 denotes that the heat pump is available to operate during that time period. This schedule may be used to completely disable the heat pump (and all of its terminal units) as required. If this field is blank, the unit is enabled the entire simulation.
+
+#### Field: Zone Terminal Unit List Name
+
+This alpha field defines the name of the zone terminal unit list. The name specified here should match the name of a valid ZoneTerminalUnitList object. In addition, each name specified in this list should match the name of a valid ZoneHVAC:TerminalUnit: VariableRefrigerantFlow object. All terminal units connected to this heat pump must be listed in this ZoneTerminalUnitList object.
+
```

Comment:
ZoneHVAC:TerminalUnit: VariableRefrigerantFlow

1) space in name, and it should be
2) ZoneHVAC:TerminalUnit:VariableRefrigerantFlow:FluidTemperatureControl